### DISCUSSION TITLE
제페토 월드 방만들기가 동작하지 않아요...

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#199] [5. Q&A]
이곳에서 작성된 멀티플레이 스크립트 샘플을 넣어서 작성했는데
멀티플레이가 가능한것까진 확인이 되었습니다
하지만 방을 만들어도 다른사람에게 전혀 보이지 않습니다 

딱히 스크립트를 고친건 전혀 없는데 왜 안되나요? 멀티플레이는 되는데 방만드는게 안되는 이유는 뭔가요?

**Comment by moondory77-naverz** [comment]
안녕하세요. 
문의주시는 내용의 범위를 조금만 더 구체화해주시면, 적확한 도움을 드릴 수 있도록 하겠습니다.
방을 만드는게 안된다는 것이, 정확히 어떤 상황을 말씀주시는 걸까요?

**Reply by JeongYire** [reply]
말 그대로입니다. 제페토 앱내에서 제 프로필에 들어가서 제가 만든 월드를 플레이하기전에 그 옆에 버튼을 눌러서 방 목록을 볼수 있지 않나요? 그 목록의 아래에서 방만들기 버튼이 있는데, 그 기능을 이용해 방을 만들어도 다른사람에게 노출되지 않습니다. 비공개로 한것도 아니구요

**Comment by cheolhui-lee** [comment]
안녕하세요.
해당 내용은 현재 서비스 안정화 작업 과정 중에 잠깐 기능을 멈춰둬서 발생한 내용으로 보입니다.
곧 재개될 수 있도록 준비하겠습니다.
불편을 끼쳐드려 죄송합니다.

**Reply by JeongYire** [reply]
감사합니다. 혹시 이런 안정화 작업이 언제 시작한다던지, 언제 끝난다던지, 정보를 게시해 두시는 곳이 있을까요? 있다면 앞으로 문제가 생겼을시 참고하도록 하겠습니다

**Reply by JeongYire** [reply]
혹시 아직 방만들기 기능이 멈춰져있나요? 제 경우엔 아직도 방만들기가 안됩니다... 
멀티플레이 기능은 제대로 작동하였습니다

**Reply by moondory77-naverz** [reply]
안녕하세요. 
우선 월드 개발 기능에서 몇가지 피쳐들 중, 추가 직후에
최적화 혹은 안정화와 관련된 이유로, 기능사용이 중지되는 사례가 있습니다.
(기능의 범위가 크리티컬 하다고 판단되는 사례의 경우, 해당 discussion을 통해 안내드리고 있습니다.)
우선, 이러한 맥락에서 양해를 부탁드리겠습니다.

말씀주신것 처럼, 각각의 피쳐에 대한 상황의 변동을 쉽게 알려드릴 수 있는 방법을 추가로 마련할 수 있도록 하겠습니다.

우선 현재 상황은, 
방 만들기 후, 현재 방 목록(빠른입장)에 노출되지 않는 이슈인데요.
(방 만들기 기능 정상작동 / 방 만들기 이후, 초대를 통해 입장 가능)

해당 기능이 재개된 이후, 다시 안내 드리겠습니다.
감사합니다.

**Reply by JeongYire** [reply]
혹시 방목록에서 노출되지않는 현상은 고쳐졌나요?
